src/metabase/util/embed.clj

```diff
@@ -100,7 +101,10 @@
                   {:leeway 60})
       ;; if `jwt/unsign` throws an Exception rethrow it in a format that's friendlier to our API
       (catch Throwable e
-        (throw (ex-info (.getMessage e) {:status-code 400}))))))
+        (if (str/starts-with? (ex-message e) "Token is expired")
+           (throw (ex-info (tru "Token is expired. This is not returned as json with the error code")
```

Comment:
This gets returned as string in the body, I need the json with the error code

Reply:
I have a hunch this could be due to whatever is catching the exception checking the :status-code, and assuming that this is an internal error we don't want to expose ex-data for if it's missing or a 500.

Reply:
Maybe try adding `:status-code 401`?